packages/tree-extension/src/index.ts

```diff
@@ -185,7 +185,7 @@ const notebookTreeWidget: JupyterFrontEndPlugin<INotebookTree> = {
 
     if (manager) {
       const running = new RunningSessions(manager, translator);
-      running.id = 'jp-running-sessions';
+      running.id = 'jp-running-sessions-tree';
```

Comment:
Looks like the following selector should also be updated in the UI tests:

https://github.com/jupyter/notebook/blob/ce9c16a01d3c80401096ccb09a8954f2b40d95de/ui-tests/test/tree.spec.ts#L43

Reply:
Thanks